src/main/groovy/org/pih/warehouse/api/PutawayTaskAdapter.groovy

```diff
@@ -129,4 +133,12 @@ class PutawayTaskAdapter {
             default: return OrderStatus.PENDING
         }
     }
+
+    static OrderItemStatusCode toOrderItemStatusCode(PutawayTaskStatus putawayTaskStatus) {
+        switch (putawayTaskStatus) {
+            case PutawayTaskStatus.COMPLETED: return OrderItemStatusCode.COMPLETED
+            case PutawayTaskStatus.CANCELED: return OrderItemStatusCode.CANCELED
+            default: return OrderItemStatusCode.PENDING
```

Comment:
If I'm not mistaken, I think we need to add two more status codes (STARTED, IN_PROGRESS) to handle those two states, right? When we were using the OrderStatus, I co-opted existing status codes, but now we can just add new ones because there's not enough specificity in the current status set.
```
enum OrderItemStatusCode {
    PENDING(1),
    STARTED(2),
    IN_PROGRESS(3),
    COMPLETED(4),   // Meaning depends on context i.e. picked, putaway, delivered, received
    CANCELED(5),
    BACKORDER(6)
...
```